doc/man3/flux_request_decode.adoc

```diff
@@ -22,7 +22,7 @@ SYNOPSIS
 
  int flux_request_decode_raw (const flux_msg_t *msg,
                               const char **topic,
-                              void *data, int *len);
+                              void **data, int *len);
```

Comment:
I should temper that comment with the observation that now it doesn't seem all that wonderful an idea.  What do others think?